module-ballerina-openapi/Package.md

```diff
@@ -1,90 +1,144 @@
+##### Modes
+If you want to generate a Service only, you can set the mode as `service` in the OpenAPI tool.
 
-### Client stub from OpenAPI
-`ballerina openapi gen-client 
-    [moduleName]:clientName 
-    openapi-contract 
-    -o[output directory name]`
-    
-Generates a Ballerina client stub for the service defined in a OpenAPI file.
+```bash
+bal openapi -i <openapi-contract-path> --mode service [(-o|--output) output file path]
+```
 
+If you want to generate a Client only, you can set the mode as  `client` in the OpenAPI tool. 
 This client can be used in client applications to call the service defined in the OpenAPI file.
 
-### Service to OpenAPI export
-`ballerina openapi gen-contract 
-    [moduleName]:serviceName 
-    [-i: ballerinaFile] 
-    [-o: contractFile] 
-    [-s: skip-bind]`
-
-Export the Ballerina service to a definition of OpenApi Specification 3.0.
-For the export to work properly, the input Ballerina service should be defined using basic service and resource level HTTP annotations.
-
-### Client stub for service
-Generates a Ballerina client stub to communicate with a Ballerina service.
-
-All endpoint(s) that are used for client stub generation should be marked with the `@openapi:ClientEndpoint` annotation. If not, there might be errors during client stub generation. Endpoints that are not marked with this annotation are not picked for client stub generation.
-The `@openapi:ClientConfig { generate: true }` annotation is used to enable or disable client stub generation per service.
+```bash
+bal openapi -i <openapi-contract-path> --mode client
+               [(-o|--output) output file path]
+```
 
-## Samples
-### Mock service from OpenAPI
-`ballerinna openapi gen-service helloworld:helloService hello_service.yaml`
+### Ballerina to OpenAPI
+#### Service to OpenAPI Export
+```bash
+bal openapi -i <ballerina-file-path> 
+               [(-o|--output) output openapi file path]
+```
+Export the Ballerina service to an  OpenAPI Specification 3.0 definition. For the export to work properly, 
+the input Ballerina service should be defined using the basic service and resource-level HTTP annotations.
+If you need to document an OpenAPI contract for only one given service, then use this command.
+```bash
+    bal openapi -i <ballerina-file-path> (-s | --service) <service-name>
+```
 
-### Client stub from OpenAPI
-`ballerina openapi gen-client hello_client hello_service.yaml`
+### Samples for OpenAPI Commands
+#### Generate Service and Client Stub from OpenAPI
+```bash
+    bal openapi -i hello.yaml
+```
 
-### OpenAPI from service
-`ballerina openapi gen-contract hello_service.bal`
+This will generate a Ballerina service and client stub for the `hello.yaml` OpenAPI contract named `hello-service` and client named `hello-client`. The above command can be run from anywhere on the execution path. 
```

Comment:
```suggestion
This will generate a Ballerina service and client stub for the `hello.yaml` OpenAPI contract named `hello-service` and client named `hello-client`. The above command can be run from within anywhere on the execution path. 
```